package.json

```diff
@@ -17,10 +17,15 @@
     "stringify": "^5.2.0"
   },
   "scripts": {
-    "build": "mkdir -p addon/compiled && google-closure-compiler addon/scripts/sjcl.js addon/scripts/config.js addon/scripts/crypto.js addon/scripts/tokens.js addon/scripts/browserUtils.js addon/scripts/background.js addon/scripts/listeners.js addon/scripts/keccak.js > addon/compiled/bg_compiled.js",
-    "test": "mkdir -p addon/compiled && google-closure-compiler addon/scripts/sjcl.js addon/scripts/config.js addon/scripts/crypto.js addon/scripts/tokens.js addon/scripts/browserUtils.js addon/scripts/background.js addon/scripts/keccak.js > addon/compiled/test_compiled.js && jest",
-    "test:quick": "jest",
-    "test:watch": "mkdir -p addon/compiled && google-closure-compiler addon/scripts/sjcl.js addon/scripts/config.js addon/scripts/crypto.js addon/scripts/tokens.js addon/scripts/browserUtils.js addon/scripts/background.js addon/scripts/keccak.js > addon/compiled/test_compiled.js && jest --watch",
+    "all": "yarn build:all && yarn test:all",
+    "build:all": "yarn build:sjcl && yarn build:ext",
+    "test:all": "yarn test:sjcl && yarn test:ext",
+    "build:ext": "mkdir -p addon/compiled && google-closure-compiler src/crypto/sjcl/sjcl.js src/ext/config.js src/ext/crypto.js src/ext/tokens.js src/ext/browserUtils.js src/ext/background.js src/ext/listeners.js src/crypto/keccak/keccak.js > addon/compiled/bg_compiled.js",
+    "test:ext": "mkdir -p addon/compiled && google-closure-compiler src/crypto/sjcl/sjcl.js src/ext/config.js src/ext/crypto.js src/ext/tokens.js src/ext/browserUtils.js src/ext/background.js src/crypto/keccak/keccak.js > addon/compiled/test_compiled.js && jest",
+    "test:ext-quick": "jest",
+    "build:sjcl": "cd src/crypto/sjcl && ./configure --with-ecc --with-codecBytes && make sjcl.js",
```

Comment:
Unnecessary code can be stripped by passing `--without-all` flag. However, there is a false dependency between `ecc`  depending on `convenience`  module. 
Verify whether the following configuration is adequate for the project.
````
 ./configure --without-all --with-ecc --with-convenience --with-codecBytes
````